pkg/es/writer/esBulkHandler.go

```diff
@@ -196,22 +229,30 @@ func HandleBulkBody(postBody []byte, ctx *fasthttp.RequestCtx, rid uint64, myid
 						}
 					}
 				} else {
-					err := ProcessIndexRequest(rawJson, tsNow, indexName, uint64(numBytes),
-						false, localIndexMap, myid, rid, idxToStreamIdCache,
-						cnameCacheByteHashToStr, jsParsingStackbuf[:])
+					ple := plePool.Get().(*writer.ParsedLogEvent)
+					defer plePool.Put(ple)
```

Comment:
not 100% sure about this (need to read the language) spec: 
if we put all the ple one by one for defer, it will go on the stack and the stack size might grow.
so do we need to put these in some array and then have just one defer call before the for loop, that would walk through the array and do the `Puts` 
(btw this does not need to be addressed in this PR, just a thought)

Reply:
Good point; I think we will want to do that